test/e2e/apps/disruption.go

```diff
@@ -335,3 +361,24 @@ func createReplicaSetOrDie(cs kubernetes.Interface, ns string, size int32, exclu
 	_, err := cs.AppsV1().ReplicaSets(ns).Create(rs)
 	framework.ExpectNoError(err, "Creating replica set %q in namespace %q", rs.Name, ns)
 }
+
+func locateARunningPod(cs kubernetes.Interface, ns string) (pod v1.Pod, err error) {
```

Comment:
Conventionally, we prefer to pass around pointers to Pods and other API objects. Shallow copies cause unexpected semantics since there are often nested pointers.

Reply:
changing to pass the pod by pointer